<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Индекс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="IndexlineEdit">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>28</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Вводится только для удаления и замены</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Название книги</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NamelineEdit">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>28</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Вводится для книг и печатных изданий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Автор книги</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AuthorlineEdit">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>28</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Вводится для книг и печатных изданий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Год издания</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="YearlineEdit">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>28</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Вводится для книг и печатных изданий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Количество страниц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PagelineEdit">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>28</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Вводится для печатных изданий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Издательство</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PublisherlineEdit">
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>28</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Вводится для печатных изданий</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ShowButton">
        <property name="text">
         <string>Отбразить всё</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ReplaceButton">
        <property name="text">
         <string>Заменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FindButton">
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LoadButton">
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="ClearButton">
        <property name="text">
         <string>Очитска</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InfoButton">
        <property name="text">
         <string>Информация о программе</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExitButton">
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
